lib/tortoise311/connection/inflight.ex

```diff
@@ -361,17 +361,12 @@ defmodule Tortoise311.Connection.Inflight do
   # Assign a random identifier to the tracked package; this will make
   # sure we pick a random number that is not in use
   defp assign_identifier(%{identifier: nil} = package, pending) do
-    case :crypto.strong_rand_bytes(2) do
-      <<0, 0>> ->
-        # an identifier cannot be zero
-        assign_identifier(package, pending)
-
-      <<identifier::integer-size(16)>> ->
-        unless Map.has_key?(pending, identifier) do
-          {:ok, %{package | identifier: identifier}}
-        else
-          assign_identifier(package, pending)
-        end
+    identifier = :rand.uniform(65535)
```

Comment:
I think this needs a comment that the return value is guaranteed to be >= 1 and <= 65535. I.e., never 0 so that doesn't need to be checked.